{"metadata":{"role":"symbol","externalID":"s:8SotoCore16AWSResponseErrorV7contextAA15AWSErrorContextVSgvp","fragments":[{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"context","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:8SotoCore15AWSErrorContextV","text":"AWSErrorContext","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"roleHeading":"Instance Property","symbolKind":"property","modules":[{"name":"SotoCore"}],"title":"context"},"abstract":[{"text":"additional context information related to the error","type":"text"}],"schemaVersion":{"major":0,"patch":0,"minor":3},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"platforms":["Linux"],"languages":["swift"]}]}],"identifier":{"url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError"]]},"variants":[{"paths":["\/documentation\/sotocore\/awsresponseerror\/context"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","sections":[],"references":{"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError/context":{"type":"topic","url":"\/documentation\/sotocore\/awsresponseerror\/context","role":"symbol","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"context"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"AWSErrorContext","preciseIdentifier":"s:8SotoCore15AWSErrorContextV"},{"kind":"text","text":"?"}],"kind":"symbol","title":"context","abstract":[{"type":"text","text":"additional context information related to the error"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError\/context"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSErrorContext":{"url":"\/documentation\/sotocore\/awserrorcontext","title":"AWSErrorContext","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSErrorContext","abstract":[{"type":"text","text":"Additional information about error"}],"kind":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"AWSErrorContext"}],"navigatorTitle":[{"kind":"identifier","text":"AWSErrorContext"}],"role":"symbol","type":"topic"},"doc://codes.soto.soto-core/documentation/SotoCore":{"kind":"symbol","type":"topic","role":"collection","url":"\/documentation\/sotocore","abstract":[{"type":"text","text":"The core framework for Soto, a Swift SDK for AWS"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","title":"SotoCore"},"doc://codes.soto.soto-core/documentation/SotoCore/AWSResponseError":{"url":"\/documentation\/sotocore\/awsresponseerror","title":"AWSResponseError","kind":"symbol","abstract":[{"type":"text","text":"Response Error type returned by Soto if the error code is unrecognised"}],"identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSResponseError","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AWSResponseError","kind":"identifier"}],"navigatorTitle":[{"text":"AWSResponseError","kind":"identifier"}],"role":"symbol","type":"topic"}}}